TACC/Core-Styles
# feat: bootstrap5 border spinner

## Overview / Changes

1. Add snippet of Bootstrap 5 to provide "[Border spinner](https://getbootstrap.com/docs/5.0/components/spinners/#border-spinner)".
2. Add TACC-specific tweaks for Bootstrap 5 Border spinner.
3. Support Bootstrap 5 Border spinner inside a TACC button.[^1]

[^1]: For https://github.com/tapis-project/authenticator/pull/__.

## Related

- required by https://github.com/tapis-project/authenticator/pull/__
- requires #357

## Testing

1. Open http://localhost:3000/components/detail/bootstrap5--border-spinner.
2. See spinner, spinner in Bootstrap button, and spinner in TACC button.
3. See spinner spins.

## UI

https://github.com/user-attachments/assets/9e97a76f-6274-42a4-87c3-f30f7b5ec063
